algorithms/unit_tests/CMakeLists.txt

```diff
@@ -153,6 +153,18 @@ foreach(Tag Threads;Serial;OpenMP;Cuda;HPX;HIP;SYCL;OpenMPTarget)
       list(APPEND STDALGO_SOURCES_E Test${Name}.cpp)
     endforeach()
 
+    # ------------------------------------------
+    # std team Q
+    # ------------------------------------------
+    set(STDALGO_TEAM_SOURCES_Q)
+    foreach(Name
+  StdAlgorithmsCommon
+  StdAlgorithmsTeamInclusiveScan
+  StdAlgorithmsTeamTransformInclusiveScan
+  )
```

Comment:
Again, weird indentation. You used tabs before.

Reply:
fixed